src/vmm/intel/vmx.c

```diff
@@ -90,6 +90,7 @@
 	 PROCBASED2_RDRAND_EXITING | \
 	 PROCBASED2_ENABLE_INVPCID /* FIXME */ | \
 	 PROCBASED2_RDSEED_EXITING | \
+	 PROCBASED2_ENABLE_XSAVES_XRSTORS | \
```

Comment:
As seen in #192 I was having similar `vmx_set_ctlreg: cap_field: 2 bit: 20 unspecified don't care` issues.

Here and below I followed the example of #174 and disabled the bits that would show up in the errors.

I have no clue if this is a good or bad thing, but it makes it work for me in the end. ^^;

Reply:
This is an unfortunate problem to have and I'm not sure this ever growing list of flags is the best way to fix this going forwards. Since this is unrelated to macOS Big Sur support, do you think you could pull this change out of the PR and just land the rest? We can then investigate how to fix this more permanently.

Reply:
will do!